catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -274,8 +264,6 @@
                             quartz-jobs,
                             c3p0,
                             terracotta-toolkit-runtime-ee,
```

Comment:
SSO itests fail and CatalogSearchUI tests fail to setup as well. If it sounds acceptable to everyone here, push forward with this PR and write another ticket to look into this `terracotta-toolkit-runtime-ee` dependency.